lib/Pod/To/HTML.pm

```diff
@@ -387,7 +386,7 @@ multi sub node2html(Pod::Block::Para $node) {
 
 multi sub node2html(Pod::Block::Table $node) {
     Debug { note colored("Table node2html called for ", "bold") ~ $node.gist };
-    my @r = '<table class="pod-table">';
+    my @r = $node.config<class>??'<table class="pod-table '~$node.config<class>~'">'!!'<table class="pod-table">';
```

Comment:
Hm… not sure if I like the fact that you can attempt injection attacks through this… Do we have any other places where this is possible?

Reply:
I'm not sure how to fix that...